init_setup.sh

```diff
@@ -111,15 +131,15 @@ docker create \
 -e SPRING_PROFILES_ACTIVE="prod" \
 -p $rest_port:8080 \
 --network $networkname \
---name $restname filefighter/rest:latest >/dev/null 2>&1
+--name $restname filefighter/rest:$restVersion >/dev/null 2>&1
```

Comment:
using >/dev/null 2>&1 hides all the download times, would be nice to get some feedback that it is doing something.

Reply:
maybe there will be a config key in the future something like `show_debug` or `silent_mode`